### PR TITLE
feat: Unleash the 20.10 horde

### PR DESCRIPTION
**Pull request by mmstick**
After ISOs are released, and we're ready to unleash desktop notifications for release upgrades.

### CODE REVIEW

File: changelogs/20.10 [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+** WiFi settings now show a QR code for sharing your Wi-Fi Hotspot with mobile devices
+** Improved fingerprint reader support
+** Option to show battery percentage
+** Screnshot and Sound Recorder had a visual refresh
+** Photos gained a new image filter, Trencin, which is similar to Instagram's Clarendon
+** Improved responsiveness of GNOME Maps
+** GNOME Clocks now allows defining a snooze and alarm duration
+** New parental controls feature after adding child accounts
+** In Wayland sessions, GNOME Shell can now drive multiple monitors with different refresh rates
+** In Wayland sessions, Mutter will now bypass the compositor when possible
```
Comment (brs17) [inline changelogs/20.10:32]: Do we need to include these if we don't ship Wayland by default?

File: changelogs/20.10 [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+** The session panel menu now has a new Restart option
+** The Application Grid is now more compact, and scales based on screen dimensions and resolution.
+** Improved responsiveness, less stuttering, and better animations
+** WiFi settings now show a QR code for sharing your Wi-Fi Hotspot with mobile devices
+** Improved fingerprint reader support
+** Option to show battery percentage
+** Screnshot and Sound Recorder had a visual refresh
+** Photos gained a new image filter, Trencin, which is similar to Instagram's Clarendon
+** Improved responsiveness of GNOME Maps
+** GNOME Clocks now allows defining a snooze and alarm duration
```
Comment (brs17) [inline changelogs/20.10:29]: Do we ship GNOME Clocks out of the box? If we don't, I'm not sure we need to mention that here.

Comment (TechnologyMan101) [reply changelogs/20.10:29]: GNOME Clocks, GNOME Maps, Photos, and Sound Recorder are not shipped by default.

Screenshot is spelled wrong.

**Comment by jacobgkau** [comment]
Clicking on the notification still brings me to the "About" page and not the "OS Upgrade" page. Running `gnome-control-center upgrade` in a terminal brings me directly to the "OS Upgrade" page. Can we make the notification open the correct page?

**Comment by mmstick** [comment]
Yeah sure

**Review by jackpot51 — APPROVED** [review]

File: changelogs/20.10 [hunk 3]
```diff
@@ -0,0 +1,29 @@
+* System apt sources have been converted to DEB822 sources format. Repoman was updated to take advantage of this in order to support defining mirrors for system sources.
+
+* System76 Power now supports a compute graphics mode. This is useful to prevent any applications from accessing it for display output, but leaving it available for CUDA and OpenCL applications.
+
+* NVIDIA driver updates enabled Reverse PRIME support. This allows for external displays to be used while on Intel graphics in Hybrid graphics mode.
+
+* Redesign of keyboard panel in collaboration with GNOME. Adds support for managing multiple bindings for a shortcut.
+
+* Popsicle was released as a flatpak on Flathub, for the benefit of other Linux distributions.
```
Comment (brs17) [inline changelogs/20.10:9]: This doesn't really seem to directly affect Pop!_OS, no?

**Review by leviport — APPROVED** [review]

**Pull request merged**